### DISCUSSION TITLE
New Controlling plugin: Manual bookings are not added to the pdf sheet

### DISCUSSION THREAD

**Discussion by habibhaidari1** [kimai/Paid-plugins#100] [Questions & Answers] [answered]
Hello,
we are using the new controlling plugin and found a possible bug. After adding manual bookings for a specific date, we expect the overtime or undertime to be corrected by the corresponding amount in the day view. However, these values do not change. Also in the generated PDF file these manual bookings are not included.

I think that is a possible bug because we are maintaining it so that we have in the PDF a working time corrected by the corresponding value from the manual booking. will this possibly be adjusted in the future?

**Comment by kevinpapst** [comment, marked as answer]
The Roadmap shows all planned features: https://www.kimai.org/store/controlling.html

And the PDF will definitely get more love.

- Monthly contract PDF
    - (2) Include: total working time sum
    - (2) Include: total holiday sum
    - (3) Include: holidays at start of month sum
    - (3) Include: start of month working-time sum


But as of today I have no plans to integrate manual bookings on the daily view. 
The normal use-case is adding or deducting a large amount of hours from a user account.
There will likely be a second page, that will list the manual bookings.

But whatever I am writing here now is subject to change, once I get to that point.

**Reply by habibhaidari1** [reply]
That's good to know. Thank you for the Answer :)

**Comment by Slomo17** [comment]
maybe not that related to the topic, but i don't want to open an new just for this question.
Regarding more love for the PDFs, i would like to send each employee a mail with the monthly pdf at the end of the month, is such a feature on the roeadmap, or as an alternative a way to download the pdf for all user at once (maybe via the api), right now its a hassle to close and download the pdf for every employee individual.
